posthog/api/notebook.py

```diff
@@ -250,8 +250,8 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                 queryset = queryset.filter(
                     last_modified_at__lt=relative_date_parse(request.GET["date_to"], self.team.timezone_info)
                 )
-            elif key == "s":
-                queryset = queryset.filter(title__icontains=request.GET["s"])
+            elif key == "search":
```

Comment:
I'll be very disappointed in myself if this doesn't cause any backend tests to fail 🤣